engines/ultima/ultima8/graphics/avi_player.h

```diff
@@ -35,7 +35,13 @@ namespace Ultima8 {
 
 class AVIPlayer : public MoviePlayer {
 public:
-	AVIPlayer(Common::SeekableReadStream *rs, int width, int height, const byte *overridePal);
+	//!
```

Comment:
I think 99% of the doxygen is using `/**` instead of `//!` - might be a good idea to unify this.

Reply:
Agreed - though the ultima8 engine currently uses this style everywhere already so I'll leave that for now in the engine code, and just fix the ones that are in the common code.